sydent/http/servlets/replication.py

```diff
@@ -163,10 +165,14 @@ def _async_render_POST(self, request):
             request.finish()
             return
 
+        now_ms = int(time.time() * 1000)
+
         for originId, inviteToken in invite_tokens.items():
             tokensStore.storeToken(inviteToken['medium'], inviteToken['address'], inviteToken['room_id'],
                                 inviteToken['sender'], inviteToken['token'],
-                                originServer=peer.servername, originId=originId, commit=False)
+                                originServer=peer.servername, originId=originId,
+                                valid_until_ts=now_ms + self.sydent.invites_validity_period if self.sydent.invites_validity_period else None,
```

Comment:
```suggestion
                                valid_until_ts=now_ms + self.sydent.invites_validity_period if self.sydent.invites_validity_period is not None else None,
```

this is getting unwieldy so it may be best to factor it out to a local variable.

Reply:
> this is getting unwieldy so it may be best to factor it out to a local variable.

Agreed